### ISSUE TITLE
dry-run option throw a error

### ISSUE THREAD

**Issue by rkmax** [db-migrate/node-db-migrate#259]
when I run `db-migrate up|down --dry-run` I get
```
[INFO] dry run
select version() as version
SHOW search_path
[ERROR] TypeError: Cannot read property 'rows' of undefined
```

**Comment by wzrdtales** [comment]
@rkmax Yes, thanks for reporting.

Those functions need to be executed through `all` as they must run even if dryRun is active. Patch is incoming.

**Comment by wzrdtales** [comment]
Bugfix also published in maintainedTesting, v0.9.x follows

**Comment by rkmax** [comment]
Thanks @wzrdtales. Man you're a beast
